<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importSecretKeys</class>
 <widget class="QDialog" name="importSecretKeys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
color: rgb(0, 0, 0);
background-color: rgb(250, 250, 250);
font-family: Poppins;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
color: rgb(0, 0, 0);
background-color: rgb(250, 250, 250);
font-family: Poppins;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="m_viewKey">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);
font-size: 12px;
font-family: Poppins;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_spendKey">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);
font-size: 12px;
font-family: Poppins;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt;
border-right: 0px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Secret Spend Key&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt;
border-right: 0px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Wallet Path&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt;
border-right: 0px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Secret View Key&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pathEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);
font-size: 12px;
font-family: Poppins;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="m_selectPathButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/images/openwallet.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_cancelButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font-size: 13px;
font-family: Poppins;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_okButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;
</string>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
font-size: 20px;
font-family: Poppins;</string>
    </property>
    <property name="text">
     <string>Import Secret Keys</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-right: 0px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Restore Your Wallet With The Secret Spend Key &amp;amp; View Key&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>m_viewKey</zorder>
   <zorder>m_spendKey</zorder>
   <zorder>m_pathEdit</zorder>
   <zorder>m_selectPathButton</zorder>
   <zorder>m_cancelButton</zorder>
   <zorder>m_okButton</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_selectPathButton</sender>
   <signal>clicked()</signal>
   <receiver>importSecretKeys</receiver>
   <slot>selectPathClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>importSecretKeys</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>importSecretKeys</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
